src/device/ping-time-linux.cpp

```diff
@@ -0,0 +1,17 @@
+#include <ping-time.h>
+
+#include <chrono>
+#include <unistd.h>
+
+int PingTime::timeMs() {
+  // std::chrono::time_point
+  auto now = std::chrono::high_resolution_clock::now();
```

Comment:
does not use auto if the type is not explicit.

Reply:
this function is not safe it can changes or go _back to the future_, use steady_clock

Reply:
> does not use auto if the type is not explicit.

I don't understand what your suggestion is here. this `auto` line comes directly from the reference documentation: https://en.cppreference.com/w/cpp/chrono

Reply:
It's an example and not necessary something that someone need to maintain. 
It's not explicit the types that each function return, so auto is not a good thing to use here.